src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -61,13 +61,33 @@ export const mapPscToBeneficialOwnerGov = (psc: CompanyPersonWithSignificantCont
     is_service_address_same_as_principal_address: isSameAddress(service_address, principal_address) ? yesNoResponse.Yes : yesNoResponse.No,
     legal_form: psc.identification?.legalForm,
     law_governed: psc.identification?.legalAuthority,
-    start_date: undefined,
+    start_date: mapStartDate(psc.notifiedOn),
     is_on_sanctions_list: psc.isSanctioned === true ? yesNoResponse.Yes : yesNoResponse.No,
   };
   mapNatureOfControl(psc, result, true);
   return result;
 };
 
+const mapStartDate = (notifiedOn: string): InputDate | undefined => {
```

Comment:
Please make this makeInputDate and take a date as arg as we'll need it more generally. Move into mapper utils.